### PR TITLE
CS Fixes

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

**Comment by taylorotwell** [comment]
Are these fixers run by the php_cs_fixer CLI tool?

**Comment by GrahamCampbell** [comment]
99% of the changes are, yeh. I had to fix 2 things myself though.

File: src/Console/Command.php [hunk 1]
```diff
@@ -56,12 +58,12 @@ public function ask($question)
...
-    * Confirm the operation with the user.
-    *
-    * @param  string  $task
-    * @param string $question
-    * @return bool
-    */
+     * Confirm the operation with the user.
+     *
+     * @param  string  $task
+     * @param  string  $question
```
Comment (GrahamCampbell) [inline src/Console/Command.php]: php-cs-fixer didn't fix this line @taylorotwell. I had to do that one myself.

Comment (GrahamCampbell) [inline src/Console/Command.php]: I haven't written a fixer to do that yet, but I will do at some point. :)

**Pull request merged**